pkg/exec/exec.go

```diff
@@ -0,0 +1,40 @@
+package exec
```

Comment:
Personally, I'd prefer to have smaller packages with defined concerns. To me, it feels like we're lumping a lot into the utils package because it's there and is easy.

I went with exec because that is an accurate name for what this is doing, but I can understand the hesitation there.